### PR TITLE
Redact LDAP password if any in request trace

### LINKED ISSUE #11742: LDAP User credentials can be gathered in plain-text via mc admin trace
If I have minio configured to use LDAP as a backend for STS, observing the trace logs with **mc admin trace -v myminio** will show the received curl command including the users plain-text (unicoded) password. 
## Expected Behavior

It would be better if credentials could be blocked in trace output. If we set up https support to minio, this handles the encryption over the network to obfuscate passwords, but any admin on the minio box should not be able view passwords. If mc admin trace is meant for debugging, is it possible to have any passwords blanked? you could have something print the url used, and instead of showing LDAPPassword=thepassword, you can have LDAPPassword=AAAAAAAA or something such that a minio admin does not gain privileged information?
## Current Behavior

Running **mc admin trace -v myminio** displays the curl line used by the client, including the LDAPPassword=thepassword in plain-text
## Steps to Reproduce (for bugs)

1. Enable LDAP support
2. Run mc admin trace on the server
3. Use a client of some sort to get an STS token
4. View the trace output to see the curl request and LDAPPassword appear

## Context

This issue has security ramifications. No one but the user really should be able to view a password in plain text if it can be avoided. 
## Your Environment

* Version used (`minio --version`):
* Server setup and configuration:
* Operating System and version (`uname -a`):
Minio version RELEASE.2021-02-11T08-23-43Z
OS: RHEL7

### PR DESCRIPTION
**Pull request by poornas**
Fixes: #11742

## How to test this PR?
Need LDAP setup
Tested with unit tests here.
## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-fs.sh | :x: [more...](#581810fc-0ba3-4cbd-9e9f-abdfcd1267c9) |
| mint-erasure.sh | :x: [more...](#e8164939-aa4f-456e-a9df-36259c1bdece) |
| mint-gateway-nas.sh | :x: [more...](#5ac22f83-1d70-4472-82cf-ebd317590663) |
| mint-compress-encrypt-dist-erasure.sh | :x: [more...](#276ad886-ac5d-4d30-bfa3-f505df5921df) |
<a name="5ac22f83-1d70-4472-82cf-ebd317590663"></a>
### 11750-110e34e/mint-gateway-nas.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:32279
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 6a6412175db6:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 1 seconds
(5/15) Running awscli tests ... done in 2 minutes and 0 seconds
(6/15) Running healthcheck tests ... done in 0 seconds
(7/15) Running mc tests ... done in 34 seconds
(8/15) Running minio-dotnet tests ... done in 32 seconds
(9/15) Running minio-go tests ... done in 1 minutes and 6 seconds
(10/15) Running minio-java tests ... done in 15 seconds
(11/15) Running minio-js tests ... done in 38 seconds
(12/15) Running minio-py tests ... FAILED in 58 seconds
{
  "name": "minio-py:test_presigned_put_object_expiry",
  "status": "FAIL",
  "args": {
    "bucket_name": "minio-py-test-ce63a143-25e8-4a46-9751-d7a3b3056998",
    "object_name": "a26008d0-6a1b-4245-a114-1d0e050b3624"
  },
  "message": "Presigned put url failed to expire!",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/minio-py/tests.py\", line 126, in _call_test\n    func(log_entry, *args, **kwargs)\n  File \"/mint/run/core/minio-py/tests.py\", line 1533, in test_presigned_put_object_expiry\n    raise ValueError('Presigned put url failed to expire!')\nValueError: Presigned put url failed to expire!\n",
  "duration": 2021
}
(12/15) Running s3cmd tests ... done in 14 seconds
(13/15) Running s3select tests ... done in 3 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="581810fc-0ba3-4cbd-9e9f-abdfcd1267c9"></a>
### 11750-110e34e/mint-fs.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev8.minio.io:32366
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp a52b05e343ca:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 0 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 2 seconds
(5/15) Running awscli tests ... done in 2 minutes and 7 seconds
(6/15) Running healthcheck tests ... done in 1 seconds
(7/15) Running mc tests ... done in 37 seconds
(8/15) Running minio-dotnet tests ... done in 38 seconds
(9/15) Running minio-go tests ... FAILED in 1 minutes and 18 seconds
{
  "args": {
    "bucketName": "minio-go-test-hv9bjanca1b9q3sn",
    "objectName": "cvrch00no626efum2g1md6shkei62a"
  },
  "duration": 14195,
  "error": "read tcp 172.17.0.4:57980->72.28.97.61:32366: read: connection reset by peer",
  "function": "GetObject(bucketName, objectName)",
  "message": "CopyN failed",
  "name": "minio-go: testSSES3EncryptedGetObjectReadSeekFunctional",
  "status": "FAIL"
}
(9/15) Running minio-java tests ... done in 27 seconds
(10/15) Running minio-js tests ... done in 45 seconds
(11/15) Running minio-py tests ... done in 1 minutes and 17 seconds
(12/15) Running s3cmd tests ... done in 14 seconds
(13/15) Running s3select tests ... done in 3 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="e8164939-aa4f-456e-a9df-36259c1bdece"></a>
### 11750-110e34e/mint-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev8.minio.io:31886
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fdf89842d708:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 1 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 41 seconds
(4/15) Running aws-sdk-ruby tests ... done in 3 seconds
(5/15) Running awscli tests ... done in 1 minutes and 15 seconds
(6/15) Running healthcheck tests ... done in 1 seconds
(7/15) Running mc tests ... done in 36 seconds
(8/15) Running minio-dotnet tests ... done in 48 seconds
(9/15) Running minio-go tests ... FAILED in 1 minutes and 58 seconds
{
  "args": {
    "bucketName": "minio-go-test-x5jujveycbxluln0",
    "objectName": "2k6zsfvz0td55pqqcsipnhg3xzyq43"
  },
  "duration": 17383,
  "error": "read tcp 172.17.0.8:48494->72.28.97.61:31886: read: connection reset by peer",
  "function": "GetObject(bucketName, objectName)",
  "message": "CopyN failed",
  "name": "minio-go: testSSES3EncryptedGetObjectReadSeekFunctional",
  "status": "FAIL"
}
(9/15) Running minio-java tests ... done in 56 seconds
(10/15) Running minio-js tests ... done in 51 seconds
(11/15) Running minio-py tests ... done in 2 minutes and 17 seconds
(12/15) Running s3cmd tests ... done in 19 seconds
(13/15) Running s3select tests ... done in 4 seconds
(14/15) Running security tests ... done in 0 seconds

Executed 14 out of 15 tests successfully.
```
<a name="276ad886-ac5d-4d30-bfa3-f505df5921df"></a>
### 11750-110e34e/mint-compress-encrypt-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev4.minio.io:31200
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 838e413091d9:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 2 seconds
(2/15) Running aws-sdk-java tests ... done in 1 seconds
(3/15) Running aws-sdk-php tests ... done in 43 seconds
(4/15) Running aws-sdk-ruby tests ... done in 4 seconds
(5/15) Running awscli tests ... FAILED in 32 seconds
{
  "name": "awscli",
  "duration": 2846,
  "function": "aws --endpoint-url http://minio-dev4.minio.io:31200 s3api copy-object --bucket awscli-mint-test-bucket-6325 --key datafile-1-kB-copy --copy-source awscli-mint-test-bucket-6325/datafile-1-kB\n",
  "status": "FAIL",
  "error": "Hash mismatch expected 084e1383b70fb0c51acc680fef370023, got ac57de7156d7fc25ac1a65f81fa3989b"
}
(5/15) Running healthcheck tests ... done in 0 seconds
(6/15) Running mc tests ... done in 46 seconds
(7/15) Running minio-dotnet tests ... done in 45 seconds
(8/15) Running minio-go tests ... FAILED in 2 minutes and 35 seconds
{
  "args": {},
  "duration": 450,
  "error": "At least one of the pre-conditions you specified did not hold",
  "function": "CopyObjectPart(destination, source)",
  "message": "CopyObjectPart call failed",
  "name": "minio-go: testUnencryptedToSSES3CopyObjectPart",
  "status": "FAIL"
}
(8/15) Running minio-java tests ... FAILED in 1 minutes and 47 seconds
{
  "name": "minio-java",
  "function": "copyObject()",
  "args": "[match etag]",
  "duration": 275,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = PreconditionFailed, message = At least one of the pre-conditions you specified did not hold, bucketName = minio-java-test-2ek7b10, objectName = minio-java-test-35hchsl-copy, resource = /minio-java-test-2ek7b10/minio-java-test-35hchsl-copy, requestId = 166AC258403C616E, hostId = fe6cc3b1-2da8-415b-9426-f7263ddd9c06)\nrequest={method=PUT, url=http://minio-dev4.minio.io:31200/minio-java-test-2ek7b10/minio-java-test-35hchsl-copy, headers=x-amz-copy-source-if-match: 71cff0a060f852067e443ad1e24ae26c-1\nx-amz-copy-source: /minio-java-test-2sksnl6/minio-java-test-35hchsl\nHost: minio-dev4.minio.io:31200\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.0.3\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20210309T191138Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20210309/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-copy-source;x-amz-copy-source-if-match;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=412, headers=Accept-Ranges: bytes\nContent-Length: 418\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nETag: \"71cff0a060f852067e443ad1e24ae26c\"\nLast-Modified: Tue, 09 Mar 2021 19:11:40 GMT\nServer: MinIO\nVary: Origin\nX-Amz-Request-Id: 166AC258403C616E\nX-Xss-Protection: 1; mode=block\nDate: Tue, 09 Mar 2021 19:11:40 GMT\n}\n >>> [io.minio.MinioClient.execute(MinioClient.java:775), io.minio.MinioClient.execute(MinioClient.java:563), io.minio.MinioClient.executePut(MinioClient.java:904), io.minio.MinioClient.copyObject(MinioClient.java:1232), FunctionalTest.testCopyObjectMatchETag(FunctionalTest.java:1850), FunctionalTest.copyObject(FunctionalTest.java:2016), FunctionalTest.runObjectTests(FunctionalTest.java:3757), FunctionalTest.runTests(FunctionalTest.java:3783), FunctionalTest.main(FunctionalTest.java:3927)]"
}
(8/15) Running minio-js tests ... done in 48 seconds
(9/15) Running minio-py tests ... done in 3 minutes and 5 seconds
(10/15) Running s3cmd tests ... FAILED in 5 seconds
{
  "name": "s3cmd",
  "duration": "2826",
  "function": "test_put_object_multipart",
  "status": "FAIL",
  "error": "WARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Too many failures. Giving up on '/mint/data/datafile-65-MB'\nERROR: \nUpload of '/mint/data/datafile-65-MB' part 1 failed. Use\n  /usr/local/bin/s3cmd abortmp s3://s3cmd-test-bucket-21067/s3cmd-test-object-25034 ca4d587b-a6a1-4292-8f80-210c734b24c5\nto abort the upload, or\n  /usr/local/bin/s3cmd --upload-id ca4d587b-a6a1-4292-8f80-210c734b24c5 put ...\nto continue the upload.\nERROR: Upload of '/mint/data/datafile-65-MB' failed too many times (Last reason: )"
}
(10/15) Running s3select tests ... done in 7 seconds
(11/15) Running security tests ... done in 0 seconds

Executed 11 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

File: cmd/http-tracer.go [hunk 1]
```diff
@@ -80,6 +81,17 @@ func (r *recordRequest) Data() []byte {
 	return logger.BodyPlaceHolder
 }
 
+var ldapPwdRegex = regexp.MustCompile("(^.*?)LDAPPassword=([^&]*?)(&(.*?))?$")
+
+// redact LDAP password if part of string
+func redactLDAPPwd(s string) string {
+	if ldapPwdRegex.MatchString(s) {
+		parts := ldapPwdRegex.FindStringSubmatch(s)
+		return parts[1] + "LDAPPassword=AAAAAAA" + parts[3]
```
Comment (vadmeste) [inline cmd/http-tracer.go]: Will we get sued if someone already uses 'AAAAAAA' as a password ?

Comment (harshavardhana) [reply cmd/http-tracer.go]: With that logic any character will be treated that way @vadmeste :P

Comment (vadmeste) [reply cmd/http-tracer.go]: > With that logic any character will be treated that way @vadmeste :P

What about 'StarTrek' instead of 'AAAAAA' ?

Comment (harshavardhana) [reply cmd/http-tracer.go]: `LDAPassword=*READACTED*` is better @poornas

File: cmd/http-tracer.go [hunk 2]
```diff
@@ -80,6 +81,17 @@ func (r *recordRequest) Data() []byte {
 	return logger.BodyPlaceHolder
 }
 
+var ldapPwdRegex = regexp.MustCompile("(^.*?)LDAPPassword=([^&]*?)(&(.*?))?$")
+
+// redact LDAP password if part of string
+func redactLDAPPwd(s string) string {
+	if ldapPwdRegex.MatchString(s) {
```
Comment (vadmeste) [inline cmd/http-tracer.go]: This is not needed, you can check with len(parts) to check if regex found a result or not.

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-zoned.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :x: [more...](#eed96c3c-b2e8-4562-90c1-b69015b03a57) |
<a name="eed96c3c-b2e8-4562-90c1-b69015b03a57"></a>
### 11750-89516b9/mint-compress-encrypt-dist-erasure.sh.log:
```
Running with
SERVER_ENDPOINT:      minio-dev7.minio.io:30807
ACCESS_KEY:           minio
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp fec7bca5031b:/mint/log /tmp/mint-logs'

(1/15) Running aws-sdk-go tests ... done in 1 seconds
(2/15) Running aws-sdk-java tests ... done in 2 seconds
(3/15) Running aws-sdk-php tests ... done in 43 seconds
(4/15) Running aws-sdk-ruby tests ... done in 4 seconds
(5/15) Running awscli tests ... FAILED in 33 seconds
{
  "name": "awscli",
  "duration": 2836,
  "function": "aws --endpoint-url http://minio-dev7.minio.io:30807 s3api copy-object --bucket awscli-mint-test-bucket-7635 --key datafile-1-kB-copy --copy-source awscli-mint-test-bucket-7635/datafile-1-kB\n",
  "status": "FAIL",
  "error": "Hash mismatch expected 084e1383b70fb0c51acc680fef370023, got ac57de7156d7fc25ac1a65f81fa3989b"
}
(5/15) Running healthcheck tests ... done in 0 seconds
(6/15) Running mc tests ... done in 43 seconds
(7/15) Running minio-dotnet tests ... done in 45 seconds
(8/15) Running minio-go tests ... FAILED in 1 minutes and 42 seconds
{
  "args": {},
  "duration": 517,
  "error": "At least one of the pre-conditions you specified did not hold",
  "function": "CopyObjectPart(destination, source)",
  "message": "CopyObjectPart call failed",
  "name": "minio-go: testUnencryptedToSSES3CopyObjectPart",
  "status": "FAIL"
}
(8/15) Running minio-java tests ... FAILED in 1 minutes and 36 seconds
{
  "name": "minio-java",
  "function": "copyObject()",
  "args": "[match etag]",
  "duration": 166,
  "status": "FAIL",
  "error": "error occurred\nErrorResponse(code = PreconditionFailed, message = At least one of the pre-conditions you specified did not hold, bucketName = minio-java-test-haf7ga, objectName = minio-java-test-uid5at-copy, resource = /minio-java-test-haf7ga/minio-java-test-uid5at-copy, requestId = 166AC6BF60FCDF4E, hostId = 30371c3b-3ad1-46df-abf5-608e37c282ba)\nrequest={method=PUT, url=http://minio-dev7.minio.io:30807/minio-java-test-haf7ga/minio-java-test-uid5at-copy, headers=x-amz-copy-source-if-match: 71cff0a060f852067e443ad1e24ae26c-1\nx-amz-copy-source: /minio-java-test-an92oa/minio-java-test-uid5at\nHost: minio-dev7.minio.io:30807\nAccept-Encoding: identity\nUser-Agent: MinIO (Linux; amd64) minio-java/8.0.3\nContent-MD5: 1B2M2Y8AsgTpgAmY7PhCfg==\nx-amz-content-sha256: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855\nx-amz-date: 20210309T203221Z\nAuthorization: AWS4-HMAC-SHA256 Credential=*REDACTED*/20210309/us-east-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-copy-source;x-amz-copy-source-if-match;x-amz-date, Signature=*REDACTED*\n}\nresponse={code=412, headers=Accept-Ranges: bytes\nContent-Length: 414\nContent-Security-Policy: block-all-mixed-content\nContent-Type: application/xml\nETag: \"71cff0a060f852067e443ad1e24ae26c\"\nLast-Modified: Tue, 09 Mar 2021 20:32:21 GMT\nServer: MinIO\nVary: Origin\nX-Amz-Request-Id: 166AC6BF60FCDF4E\nX-Xss-Protection: 1; mode=block\nDate: Tue, 09 Mar 2021 20:32:21 GMT\n}\n >>> [io.minio.MinioClient.execute(MinioClient.java:775), io.minio.MinioClient.execute(MinioClient.java:563), io.minio.MinioClient.executePut(MinioClient.java:904), io.minio.MinioClient.copyObject(MinioClient.java:1232), FunctionalTest.testCopyObjectMatchETag(FunctionalTest.java:1850), FunctionalTest.copyObject(FunctionalTest.java:2016), FunctionalTest.runObjectTests(FunctionalTest.java:3757), FunctionalTest.runTests(FunctionalTest.java:3783), FunctionalTest.main(FunctionalTest.java:3927)]"
}
(8/15) Running minio-js tests ... done in 52 seconds
(9/15) Running minio-py tests ... done in 3 minutes and 16 seconds
(10/15) Running s3cmd tests ... FAILED in 5 seconds
{
  "name": "s3cmd",
  "duration": "2665",
  "function": "test_put_object_multipart",
  "status": "FAIL",
  "error": "WARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Retrying upload of /mint/data/datafile-65-MB\nWARNING: MD5 Sums don't match!\nWARNING: Too many failures. Giving up on '/mint/data/datafile-65-MB'\nERROR: \nUpload of '/mint/data/datafile-65-MB' part 1 failed. Use\n  /usr/local/bin/s3cmd abortmp s3://s3cmd-test-bucket-6873/s3cmd-test-object-6304 31700a06-de52-4a56-9271-99150886af4e\nto abort the upload, or\n  /usr/local/bin/s3cmd --upload-id 31700a06-de52-4a56-9271-99150886af4e put ...\nto continue the upload.\nERROR: Upload of '/mint/data/datafile-65-MB' failed too many times (Last reason: )"
}
(10/15) Running s3select tests ... done in 7 seconds
(11/15) Running security tests ... done in 0 seconds

Executed 11 out of 15 tests successfully.
```
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Review by harshavardhana — APPROVED** [review]

**Comment by groucho64738** [comment]
This doesn't seem to be fixed in 2021-03-10T05-11-33Z. I still see 12:10:34.039 [400 Bad Request] sts.AssumeRoleWithLDAPIdentity test-min1:9000/?Action=AssumeRoleWithLDAPIdentity&LDAPUsername=testuser&LDAPPassword=canyoureadit%3F&Version=2011-06-15  xx.xx.xx.xx      6.367ms      ↑ 65 B ↓ 543 B

This was using the updated mc and minio binaries. I'm @KT in slack if you want me to try something else.

**Pull request merged**